payloads/library/GitBunnyGit/README.md

```diff
@@ -0,0 +1,36 @@
+# Git-Bunny-Git
+
+Author: Draxiom & audibleblink
+Version: 1.0
+
+## Description
+
+Clones the bashbunny-payloads repository and also will update an existing repository.
+If you modify your payloads from the library folder, they will be overwritten.
```

Comment:
not technically true in this case. the pull will fail because the user will have uncommited changes. It was true LibraryUpdate where I was rsyncing things.